VideoLocker/src/test/java/org/edx/mobile/discussions/DiscussionTextUtilsTest.java

```diff
@@ -152,15 +154,29 @@ private void assertSetAuthorAttributionText(TextView textView,
                 int start = output.indexOf(input.getAuthor());
                 int end = start + input.getAuthor().length();
                 SpannableString text = (SpannableString) textView.getText();
-                ClickableSpan[] spans = text.getSpans(start, end, ClickableSpan.class);
+                StyleSpan[] styleSpans = text.getSpans(start, end, StyleSpan.class);
+                ForegroundColorSpan[] colorSpans = text.getSpans(start, end, ForegroundColorSpan.class);
                 if (config.isUserProfilesEnabled()) {
-                    assertEquals(1, spans.length);
-                    assertEquals(start, text.getSpanStart(spans[0]));
-                    assertEquals(end, text.getSpanEnd(spans[0]));
-                    spans[0].onClick(textView);
+                    // Check if the author text is Bold
```

Comment:
The 'Check if' can be replaces with 'Verify that' in these comments. Also, 'Bold' shouldn't be capitalized here.